src/main/components/ResponseComponent.jsx

```diff
@@ -1,23 +1,66 @@
 import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import DataTree from './DataTree';
+
+const ResponseWrapper = styled.article`
+  background-color: #F0F3F4;
+  border-radius: 3px;
+  box-shadow: inset 
+  box-shadow: inset 0 0 1px 0 rgba(41,47,50,0.25);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1em;
+  margin-bottom: 1em;
+`
+
+const Icon = styled.span`
+  font-family: 'SF Pro Text';
+  margin-right: 1em;
+  color: ${(props) => {
```

Comment:
I like this, I've never thought to write strings this way

Reply:
Styled components are sweet 😎